Tests/DependencyInjection/MonologExtensionTest.php

```diff
@@ -292,6 +295,9 @@ public function testRavenHandlerWhenConfigurationIsWrong()
         }
     }
 
+    /**
+     * @group version1only
```

Comment:
Wouldn't it be better to detect the monolog version in the test and skip them instead of relying on the caller to exclude a group ?

Reply:
Yes, I am currently working on replacing this with a `markTestSkipped` where I look for the handler class from Monolog 1 instead. This way the test will also be more reliable when running locally without the flag